README.md

```diff
@@ -1,2 +1,2 @@
-# code-review-exercise
-practice on code review
+# youtube-videoplayer
+Learning to work with media in a simple webpage
```

Comment:
Please add some more information on your README like how to install and run the application, what technology is used etc . The README file helps the end user to smoothly run the application. Kindly check the template provided by Microverse and customize it to fit your project.